### PR TITLE
adds markdown to concept descriptions

### PR DESCRIPTION
**Pull request by steven-ferguson**
Issue #43 complete.

### CODE REVIEW

File: app/assets/stylesheets/layouts.css.scss [hunk 1]
```diff
@@ -23,3 +23,7 @@ $mercury: #d1d1d1;
 .concepts-list {
   margin-top: 20px;
 }
+
+.hiding {
```
Comment (ErinCall) [inline app/assets/stylesheets/layouts.css.scss:55]: aww it's shy :)

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -7,4 +7,23 @@ def sponsors
...
+    renderer = HTMLwithPygments.new(hard_wrap: true, filter_html: true)
+    options = {
+      autolink: true,
+      no_intra_emphasis: true,
+      fenced_code_blocks: true,
+      lax_html_blocks: true,
+      strikethrough: true,
+      superscript: true
+    }
+    Redcarpet::Markdown.new(renderer, options).render(text).html_safe
```
Comment (ErinCall) [inline app/helpers/application_helper.rb:27]: This needs tests!

Comment (steven-ferguson) [inline app/helpers/application_helper.rb:27]: Updated with integration test for markdown.

**Comment by ErinCall** [comment]
There's a failing test: [see the Travis CI build](https://travis-ci.org/code-scouts/green_mercury/builds/12895538)

What's desktop/test.png? Was that added on purpose?

**Comment by ErinCall** [comment]
Huh so that last "fix conflicts" commit doesn't look right at all. Will you revert it and merge from code-scouts/master?

**Comment by ErinCall** [comment]
There's a failing spec that doesn't look timezone related.

**Comment by ErinCall** [comment]
It's passing now!

**Pull request merged**